template/requirements/make_base.py

```diff
@@ -58,11 +58,12 @@ def as_nightly(repo: str) -> str:
     else:
         org = "scipp"
     if repo == "scipp":
-        version = f"cp{sys.version_info.major}{sys.version_info.minor}"
-        base = "https://github.com/scipp/scipp/releases/download/nightly/scipp-nightly"
-        suffix = "manylinux_2_17_x86_64.manylinux2014_x86_64.whl"
-        prefix = "scipp @ "
-        return prefix + "-".join([base, version, version, suffix])
+        return (
```

Comment:
for now, let's just start with scipp?

Reply:
Sure, we start with scipp.
But we need to put the others there as well so that the integration tests are running on latest nightly `main`, because at the moment, they are using latest released as we cannot install from github (ip address is getting throttled)